storage/innobase/srv/srv0start.cc

```diff
@@ -1231,6 +1231,20 @@ srv_open_tmp_tablespace(
 		it stays open until shutdown. */
 		if (fil_space_open(tmp_space->name())) {
 
+			if (srv_tmp_tablespace_encrypt) {
+
+				fil_space_t*		space;
+
+				space = fil_space_get(temp_space_id);
+				space->flags |= FSP_FLAGS_MASK_ENCRYPTION;
+
+				err = fil_set_encryption(space->id,
+							 Encryption::AES,
+							 NULL,
+							 NULL);
+				ut_ad(err == DB_SUCCESS);
```

Comment:
If we are asserting here that fil_set_encryption is "impossible" (so that crashing a debug build instead of printing errors, aborting server startup is reasonable) then what about release build? Maybe ut_a here?

Reply:
agree on that too